src/views/self/view_self.cpp

```diff
@@ -80,6 +81,10 @@ namespace big
 		ImGui::Checkbox("No Collision", &g->self.no_collision);
 		ImGui::Checkbox("Mobile Radio", &g->self.mobile_radio);
 
+		ImGui::Checkbox("Dance Mode", &g->self.dance_mode);
+		if (ImGui::IsItemHovered())
+			ImGui::SetTooltip("Hold Right DPAD or E to enter dance mode");
```

Comment:
Feature seems to trigger unreliably; I've gotten it to work once and when I tried to quit it, I deadlocked myself, had to suicide myself and reset the hidden hud elements.

Reply:
Maybe switching it to a button would be better? That way it can only be toggled off by using the in-game options (`ESC` for example to stop dancing).